Source/Core/Core/IOS/USB/Emulated/Skylanders/SkylanderFigure.cpp

```diff
@@ -0,0 +1,445 @@
+        *reinterpret_cast<const u8*>(decrypted.data() + areaOffset + 0x14);
+  }
+
+  return figureData;
+}
+void SkylanderFigure::SetData(FigureData* figureData)
+{
+  std::array<u8, FIGURE_SIZE> decrypted = {};
+
+  DecryptFigure(decrypted.data());
+
+  if (static_cast<Type>(figureData->normalizedType) == Type::Skylander)
+  {
+    // Only update area with lowest counter
+    u16 areaOffset = (decrypted[0x89] != (decrypted[0x249] + 1U)) ? 0x80 : 0x240;
+
+    memcpy(decrypted.data() + areaOffset + 0x3, &figureData->skylanderData.money, 2);
+    memcpy(decrypted.data() + areaOffset + 0x5A, &figureData->skylanderData.heroLevel, 2);
+    memcpy(decrypted.data() + areaOffset + 0x5, &figureData->skylanderData.playtime, 4);
+
+    {
+      memcpy(decrypted.data() + areaOffset + 0x60, &figureData->skylanderData.lastReset.minute, 1);
+      memcpy(decrypted.data() + areaOffset + 0x61, &figureData->skylanderData.lastReset.hour, 1);
+      memcpy(decrypted.data() + areaOffset + 0x62, &figureData->skylanderData.lastReset.day, 1);
+      memcpy(decrypted.data() + areaOffset + 0x63, &figureData->skylanderData.lastReset.month, 1);
+      memcpy(decrypted.data() + areaOffset + 0x64, &figureData->skylanderData.lastReset.year, 2);
+    }
+
+    {
+      memcpy(decrypted.data() + areaOffset + 0x50, &figureData->skylanderData.lastPlaced.minute, 1);
+      memcpy(decrypted.data() + areaOffset + 0x51, &figureData->skylanderData.lastPlaced.hour, 1);
+      memcpy(decrypted.data() + areaOffset + 0x52, &figureData->skylanderData.lastPlaced.day, 1);
+      memcpy(decrypted.data() + areaOffset + 0x53, &figureData->skylanderData.lastPlaced.month, 1);
+      memcpy(decrypted.data() + areaOffset + 0x54, &figureData->skylanderData.lastPlaced.year, 2);
+    }
+
+    {
+      for (u8 i = 0; i < 8; i++)
+      {
+        memcpy(decrypted.data() + areaOffset + 0x20 + (i * 2),
+               &figureData->skylanderData.nickname[i], 0x2);
+      }
+
+      for (u8 i = 0; i < 8; i++)
+      {
+        memcpy(decrypted.data() + areaOffset + 0x40 + (i * 2),
+               &figureData->skylanderData.nickname[8 + i], 0x2);
+      }
+    }
+
+    {
+      ComputeChecksum(ChecksumType::Type3, decrypted.data() + areaOffset + 0x50,
+                      decrypted.data() + areaOffset + 0xA);
+      ComputeChecksum(ChecksumType::Type2, decrypted.data() + areaOffset + 0x10,
+                      decrypted.data() + areaOffset + 0xC);
+      if (decrypted[0x89] == decrypted[0x249])
+        decrypted[areaOffset + 0x9]++;
+      else
+      {
+        decrypted[areaOffset + 0x9] += 2;
+      }
+      ComputeChecksum(ChecksumType::Type1, decrypted.data() + areaOffset,
+                      decrypted.data() + areaOffset + 0xE);
+    }
+
+    {
+      areaOffset = (decrypted[0x112] != (decrypted[0x2D2] + 1U)) ? 0x110 : 0x2D0;
+
+      if (decrypted[0x112] == decrypted[0x2D2])
+        decrypted[areaOffset + 0x2]++;
+      else
+      {
+        decrypted[areaOffset + 0x2] += 2;
+      }
```

Comment:
Remove the braces from the `else` condition.

For the future, if the `if` statement lacks braces, the `else` statement should also not have braces. If the `else` statement has more than two lines of code, then you should add braces to the `if` statement.